detectron2/data/dataset_mapper.py

```diff
@@ -73,6 +73,12 @@ def __call__(self, dataset_dict):
         dataset_dict = copy.deepcopy(dataset_dict)  # it will be modified by code below
         # USER: Write your own image loading if it's not from a file
         image = utils.read_image(dataset_dict["file_name"], format=self.img_format)
+        # To ensure bbox always remap to original image size
+        if "width" not in dataset_dict:
+            dataset_dict["width"] = image.shape[1]
+        if "height" not in dataset_dict:
+            dataset_dict["height"] = image.shape[0]
+
         utils.check_image_size(dataset_dict, image)
```

Comment:
could you implement it inside this function instead?

Reply:
hi, yuxin, which function do you mean please?

Reply:
@ppwwyyxx hi, yuxin, new commit is here: https://github.com/facebookresearch/detectron2/pull/249/commits/9374908e079947749d41f0f2c03ba49b910c93b4